app/views/state_file/questions/nj_review/edit.html.erb

```diff
@@ -6,7 +6,7 @@
   <% if current_intake.dependents.count(&:under_22?).positive? %>
     <section id="college-dependents" class="white-group">
       <div class="spacing-below-5">
-        <h2 class="text--body text--bold spacing-below-5"><%=t(".college_dependents") %></h2>
```

Comment:
[a mound of pebbles] - looks like the `h2` -> `h3` change was only on the opening tag and not on the closing one. And that seems to apply to all the changes in this PR. it might still render correctly in browsers and test drivers which would hide the issue, but we should fix it